.github/CODEOWNERS

```diff
@@ -314,6 +314,8 @@ Tasks/NodeTaskRunnerInstallerV0/    @microsoft/akvelon-build-task-team
 
 Tasks/NodeToolV0/ @microsoft/akvelon-build-task-team
 
+Tasks/NotationV0/ @microsoft/akvelon-build-task-team @JeyJeyGao @shizhMSFT
```

Comment:
No, akvelon-build-task-team is not the owner here. Please create your own github team and replace this.

Reply:
Updated to be @Azure/azure-container-registry @JeyJeyGao @Two-Hearts @shizhMSFT